File: src/test/demo/java/link.rdcn/ServerDemo.java
/**
 * @Author Yomi
 * @Description:
 * @Data 2025/7/15 10:57
 * @Modified By:
 */
package link.rdcn;

import link.rdcn.server.FairdServer;

public class ServerDemo {
    public static void main(String[] args) {
        TestProvider provider = new TestProvider();
        //根据fairdHome自动读取配置文件
        FairdServer server = new FairdServer(provider.dataProvider(),provider.authProvider(),provider.getResourcePath("tls"));
        server.start();
    }
}

File: src/test/demo/scala/link/rdcn/ServerDemo.scala
/**
 * @Author Yomi
 * @Description:
 * @Data 2025/7/22 10:49
 * @Modified By:
 */
package link.rdcn

import link.rdcn.server.FairdServer

import java.nio.file.Paths


object ServerDemo {
  def main(args: Array[String]): Unit = {
    val provider = new TestProvider
    //根据fairdHome自动读取配置文件
    val server = new FairdServer(provider.dataProvider, provider.authProvider, Paths.get(provider.getResourcePath("tls")).toString())
    server.start()
  }
}
